tree/treeplayer/src/TTreeFormula.cxx

```diff
@@ -4006,7 +4006,7 @@ T TTreeFormula::EvalInstance(Int_t instance, const char *stringStackArg[])
       }
    }
 
-   T tab[kMAXFOUND];
+   T tab[kMAXFOUND]{};
```

Comment:
This is performance critical code and initializing (most often unnecessarily) is likely to reduce performance significantly

Reply:
Ok. I can remove it. But clang tidy analyzer warns about potential pathways where random memory may be read

Reply:
Those have to be carefully looked at.  They may or may not be false positive.